### ISSUE TITLE
collect runs into network timeout

### ISSUE THREAD

**Issue by hummal** [GoogleChrome/lighthouse-ci#167]
I got the lhci server running and created a project through the wizard. Token is set.

When I launch `./node_modules/.bin/lhci collect --url=https://example.com` I got the following result:

```bash
Started a web server on port 37817...
Running Lighthouse 3 time(s) on https://example.com:37817/
Run #1...
```

That results in a network timeout error after some minutes. 

Iam not sure why lhci is creating another port on a external url. Is that the expected behaviour? There is no route to this url on my machine and Iam not the owner of example.com. So I guess there is no port 37817. 

What am I missing? 

My config:
```json{
  "ci": {
    "collect": {
      "numberOfRuns": 3,
      "settings": {
	"chromeFlags": "--disable-gpu --no-sandbox"
      },
     "staticDistDir": "./public"
    },
    "assert": {
      "preset": "lighthouse:recommended"
    },
    "upload": {
	"target": "lhci",
	"serverBaseUrl": "127.0.0.1:9001",
	"token": "f69036ae-88ec-4cb5-b2b6-a7a030b95584"
    },
    "server": {
	"port": 9001,
	"storage": {
		"sqlDatabasePath": "/root/scripts/lhci_server/lhci.db"
	}
    }
  }
}```

**Comment by patrickhulce** [comment]
tl;dr `staticDistDir` should not be used in combination with URLs that aren't files in the `staticDistDir`, pick one or the other for your case.

> Iam not sure why lhci is creating another port on a external url. 

You passed the `staticDistDir` option which tells Lighthouse CI you want to audit local static files on `localhost`, so Lighthouse CI starts a webserver on a local port that will be injected into the URLs you provide, so that you can do `http://localhost/my/article` or `http://127.0.0.1/app/pricing` and have the port be correct.

But then later when you run it you say you want to audit `https://example.com`. If you want to do that you should remove `staticDistDir`.

**Comment by hummal** [comment]
Thank you for clearing things up. It is working now. Maybe the documentation could be more clear about that.

**Comment by patrickhulce** [comment]
> Maybe the documentation could be more clear about that.

Do you have any suggestions to improve wording that led you astray or what you think should be added?

**Comment by hummal** [comment]
Thanks for replying. 

I would simply suggest to list and explain every single configuration parameter.
At the moment there are just examples and recommendations. That configuration list should be easy to find and shot to read. I guess that is what would have lead me to the right position without having to create an issue. 

Cheers and thanks for you effort
